nemo/collections/llm/quantization/utils.py

```diff
@@ -73,6 +73,8 @@ def load_with_modelopt_layer_spec(
     nemo_checkpoint_path: str,
     tensor_model_parallel_size: int = 1,
     pipeline_model_parallel_size: int = 1,
+    devices_per_node: int = 1,
```

Comment:
nit: lets name this `devices` as that's the standard name being used everywhere in nemo